superset-frontend/src/explore/components/DataTableControl.jsx

```diff
@@ -0,0 +1,91 @@
+/**
```

Comment:
this is a new file, can you please use TypeScript?

Reply:
Done